### ISSUE TITLE
[BUG] - Item Piles Currency does not work properly

### ISSUE THREAD

**Issue by Shuggaloaf** [fantasycalendar/FoundryVTT-ItemPiles#63]
**Describe the bug**
Related to issue #60 
When using an Item Pile, adding currency sometimes adds some coins but not others. Sometimes it does not add the total amount you try to add to a specific coin type. Most of the time it adds nothing at all.

When adding currency through the Item Pile Token's sheet, it works as intended for all currency types.

**To Reproduce**

https://user-images.githubusercontent.com/79543184/153634080-5d257972-7959-4320-bc7b-c08e9704f191.mp4

**Setup:**
- Foundry Version: 0.8.9
- System: Dnd5e 1.5.6
- Item Piles Version: 1.4.1
- OS: Windows 10
- Browser: Chrome/Electron

**Active modules:**
 - Item Piles
 - libwrapper
 - socketlib
 - All Other Mods Disabled

**Comment by Haxxer** [comment]
Ah it seems that you are inspecting this pile as a token, that's weird.

**Comment by Haxxer** [comment]
Do you, the GM, have a character assigned to you or something?

![image](https://user-images.githubusercontent.com/12858387/153641618-bce10331-087d-4350-95ea-752718ebc173.png)

**Comment by Haxxer** [comment]
Okay, I found the bug. Basically, when you have a selected token (or an assigned character), you can only drop currencies into the item pile that the selected character has in its inventory. HOWEVER, there's a bug currently where the "drop currencies" dialog shows that you can drop anything, but in reality, you can only drop what the actor has.

Compounding this issue, as a GM, you're not restricted by distance when inspecting piles, so when you have a default character AND you inspect a pile, you can _only_ inspect the pile as that character unless you select another character before you double click on the pile.

In short, stuff is weird, and I'll address this in the next update.

**Comment by Shuggaloaf** [comment]
>Do you, the GM, have a character assigned to you or something?

Yes I have a token assigned. Just a character i use for testing things. It sounds like that may be abnormal? Could it be causing issues like this and possibly others? 

>you can only drop currencies into the item pile that the selected character has in its inventory

Confirmed! That makes sense now as to why the first time I tried dropping 10 to each, it dropped 3 to PP, then would drop none after that. Character must have had 3 PP.

>Compounding this issue [...] when you have a default character 

So it sounds like it may be best to try not having one. I can't remember why I started doing this way back in 0.7x. It may have been something to do with vision testing but not sure. Will try disabling that and see if there's anything else it affects. 

>In short, stuff is weird, and I'll address this in the next update.

Haha no worries, glad you nailed it down to what the issue is. Loving this mod so far by the way!

**Comment by Haxxer** [comment]
Fixed in 1.4.2
